perf/tests/package.json

```diff
@@ -15,8 +15,8 @@
     "studio:dev": "cd perf/studio && SANITY_STUDIO_DATASET=dev pnpm dev"
   },
   "dependencies": {
-    "@playwright/test": "^1.41.2",
-    "@sanity/client": "^6.15.7",
+    "@playwright/test": "1.41.2",
```

Comment:
do we really want to pin to a specific version?

Reply:
Sorry. I think this is due to a bad rebase. Will fix